### ISSUE TITLE
Some emojis are not displayed. 

### ISSUE THREAD

**Issue by zeitnot** [hidakatsuya/prawn-emoji#34]
First of all, thanks for this great gem. It works really great. But I have faced a problem when it comes to display some emojis. 

When I render the pdf on Firefox all emojis are displayed nicely.

Here is a screenshot from firefox.
![image](https://user-images.githubusercontent.com/1460930/85620525-94f48f00-b66c-11ea-995d-0bd7a882f9ed.png)

So far so good. 

And now here is screenshot from Safari browser:

<img width="1594" alt="image" src="https://user-images.githubusercontent.com/1460930/85620753-e69d1980-b66c-11ea-9857-eb1f5307a747.png">

I have also opened it on Mac's default pdf viewer:

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/1460930/85620958-2fed6900-b66d-11ea-8119-cf885f36a98f.png">

Safari & Mac's Preview behave the same and they show only smiley face. Other emojis are rendered but they have problem. 
I have debuged the code and realize that prawn splits the text [here](https://github.com/prawnpdf/prawn/blob/master/lib/prawn/text/formatted/box.rb#L455) if it find unicode such as any kind of emoji.

**My prawn configuration:**

Prawn version: 2.2.2
Prawn-emoji: 3.3.0

```ruby
    font_families.update(
      'SFProDisplay' => {
        normal: Rails.root.join('app', 'assets', 'fonts', 'SFProDisplay-Regular.ttf'),
        bold: Rails.root.join('app', 'assets', 'fonts', 'SFProDisplay-Bold.ttf'),
        semi_bold: Rails.root.join('app', 'assets', 'fonts', 'SFProDisplay-Semibold.ttf'),
      },
      'DejaVuSans' => {
        normal: Rails.root.join('app', 'assets', 'fonts', 'DejaVuSans.ttf'),
        bold: Rails.root.join('app', 'assets', 'fonts', 'DejaVuSans-Bold.ttf'),
        semi_bold: Rails.root.join('app', 'assets', 'fonts', 'DejaVuSans-Bold.ttf')
      }
    )

    fill_color DEFAULT_COLOR
    font 'SFProDisplay'
    fallback_fonts ['DejaVuSans']
```

**NOTE**: If I set default font as `DejaVuSans` instead of `SFProDisplay` then all emojis are displayed. 

I could not understand the problem otherwise I could open a PR. If you guide me if there is some sort of a bug I can send a PR.

**Comment by hidakatsuya** [comment]
Thank you for your report. I have confirmed this problem.

**Comment by hidakatsuya** [comment]
@zeitnot 
I released [v4.0.0.beta.1](https://github.com/hidakatsuya/prawn-emoji/blob/master/CHANGELOG.md#400beta1). This version also includes a fix for this problem. Could you try this version?

**Comment by zeitnot** [comment]
@hidakatsuya Thanks, it works now.

**Comment by hidakatsuya** [comment]
I have released [v4.0.0](https://github.com/hidakatsuya/prawn-emoji/blob/master/CHANGELOG.md#400) which also includes the fix for this problem.
